hack/local-up-cluster.sh

```diff
@@ -1023,6 +1030,10 @@ if [[ "${START_MODE}" != "kubeletonly" ]]; then
   start_kubedashboard
 fi
 
+if [[ "${START_MODE}" != "nokubeproxy" ]]; then
+  start_kubeproxy
```

Comment:
why are we changing the order of startup here, shouldn't this still be happening before `stat_kubescheduler` above?

Reply:
In my testing it didn't affect, but I am not sure if I am missing a case which it won't work. 
I will move the check to Line 1017